server/sync/onchain.go

```diff
@@ -181,13 +182,32 @@ func (s *Syncer) getProposalTitleAndDescription(proposalID string) (string, stri
 	// Match all top-level parentheses contents: ("..." something)
 	re := regexp.MustCompile(`\("([^"]+)" string\)`)
 
-	matches := re.FindAllSubmatch(data.Response.Data, -1)
+	matches := re.FindAllString(string(data.Response.Data), -1)
```

Comment:
this still seems brittle, how this code would react if if put ` string) ("lol" string) (` in my proposal title?

I recommend to make one call per value (for example: `gno.land/r/gov/dao.GetProposal(cross,%s).Title`) and call `extractGnoStrinResponse` on the raw result to avoid potential injections